packages/api/src/chats/usecases/upload-documents-to-chat.usecase.ts

```diff
@@ -61,7 +61,7 @@ export class UploadDocumentsToChatUsecase implements Usecase {
   ) {
     if (
       chat.documents.length >= 2 ||
-      files.length + chat.documents.length >= 2
+      files.length + chat.documents.length > 2
```

Comment:
What do you think about adding the max document number to the config file?

Reply:
that makes total sense! will do